### PR TITLE
Corporate cards section in Dashboard changes

### PR DESCRIPTION
**Pull request by lakshmisuhas1996**

### CODE REVIEW

File: src/app/fyle/dashboard/stats/stats.component.html [hunk 1]
```diff
@@ -122,6 +122,128 @@
       </ion-row>
     </ion-grid>
   </div>
+
+  <div *ngIf="(isConnected$|async)" class="stats--ccc-cards-section">
```
Comment (Julias0) [inline src/app/fyle/dashboard/stats/stats.component.html:126]: dont we show a zero state in offline mode?

File: src/app/fyle/dashboard/stats/stats.component.scss [hunk 2]
```diff
@@ -154,4 +156,118 @@
...
+    border-radius: 16px;
+    width: calc(100% - 32px);
+    position: relative;
+    left: 50%;
+    transform: translate(-50%, 0px);
+    bottom: 54px;
+  }
+
+  &--ccc-card-info {
+    background: rgba(255, 255, 255, 0.12);
```
Comment (Julias0) [inline src/app/fyle/dashboard/stats/stats.component.scss:199]: colors move to colors.scss

File: src/app/fyle/dashboard/stats/stats.component.ts [hunk 3]
```diff
@@ -98,6 +101,13 @@ export class StatsComponent implements OnInit {
     );
   }
 
+  initializeCCCStats() {
+    this.cardTransactionsAndDetails$ = this.dashboardService.getCCCDetails().pipe(
+      shareReplay(1),
```
Comment (Julias0) [inline src/app/fyle/dashboard/stats/stats.component.ts:106]: do the shareReplay after the map

File: src/app/fyle/dashboard/stats/stats.component.html (hunk 1 above)
Comment (lakshmisuhas1996) [reply src/app/fyle/dashboard/stats/stats.component.html:126]: We already showing one for expenses stats card when offline....

File: src/app/fyle/dashboard/stats/stats.component.ts [hunk 4]
```diff
@@ -43,6 +44,8 @@ export class StatsComponent implements OnInit {
 
   loadData$ = new Subject();
 
+  cardTransactionsAndDetails$: Observable<BankAccountsAssigned>;
```
Comment (Julias0) [inline src/app/fyle/dashboard/stats/stats.component.ts:47]: set this to of(null) if `orgSettings.corporate_credit_card_settings.enabled` is false

**Review by Julias0 — APPROVED** [review]

**Pull request closed without merging**